portfolio/src/main/webapp/script.js

```diff
@@ -34,4 +34,22 @@ function pickRandomQuote() {
 function divMessage() {
     const quoteToDisplay = pickRandomQuote();
     document.getElementById('quote-displayer').innerText = quoteToDisplay;
+}
+
+function fetchMyName() {
+    console.log('Fetching your name...');
+    const responsePromise = fetch('/data');
+    responsePromise.then(handleResponse);
+}
+
+function handleResponse(response) {
```

Comment:
This function name is too generic. it sounds like it can process any response but in reality it's only for a name response. Same with the parameter name.